client/src/modules/employees/registry/search.modal.html

```diff
@@ -11,130 +11,140 @@
     </ol>
   </div>
 
-  <div class="modal-body" style="max-height : 400px; overflow: auto;">
-    <fieldset>
-      <legend translate>FORM.LABELS.DATE_EMBAUCHE</legend>
-      <bh-date-interval
-        date-id="embauche-date"
-        validation-trigger="ModalForm.$submitted"
-        date-from="ModalCtrl.searchQueries.dateEmbaucheFrom"
-        date-to="ModalCtrl.searchQueries.dateEmbaucheTo"
-        mode="clean">
-      </bh-date-interval>
-    </fieldset>
-
-    <br />
-
-    <fieldset>
-      <legend translate>FORM.LABELS.EMPLOYEE_DETAILS</legend>
-
-      <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.name.$invalid }">
-        <label class="control-label" translate>FORM.LABELS.NAME</label>
-        <bh-clear on-clear="ModalCtrl.clear('display_name')"></bh-clear>
-        <input type="text" class="form-control" name="display_name" ng-model="ModalCtrl.searchQueries.display_name">
-        <div class="help-block" ng-messages="ModalForm.display_name.$error" ng-show="ModalForm.$submitted">
-          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
-        </div>
+  <uib-tabset>
+    <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+
+      <div class="modal-body" style="max-height : 400px; overflow: auto;">
+        <fieldset>
+          <legend translate>FORM.LABELS.DATE_EMBAUCHE</legend>
```

Comment:
A `legend` is really big!  This could be a simple `label` to conserve space on the modal.   The same fix should be applied to all `<legend>`s on this modal.